src/pages/public_registration/PublicGeneralContent.tsx

```diff
@@ -128,104 +128,116 @@ export const PublicGeneralContent = ({ registration }: PublicRegistrationContent
         <Typography variant="h3" component="h2" gutterBottom>
           {t('registration.public_page.about_registration')}
         </Typography>
-        {alternativeTitles.length > 0 && (
-          <PublicPageInfoEntry
-            title={t('registration.description.alternative_title')}
-            content={alternativeTitles.map((title) => (
-              <Typography key={title}>{title}</Typography>
-            ))}
-          />
-        )}
-        {dateString && <PublicPageInfoEntry title={t('common.date')} content={dateString} />}
-        {language && (
-          <PublicPageInfoEntry
-            data-testid={dataTestId.registrationLandingPage.primaryLanguage}
-            title={t('common.language')}
-            content={i18n.language === 'nob' ? language.nob : language.eng}
-          />
-        )}
-        {entityDescription?.npiSubjectHeading && (
-          <PublicPageInfoEntry
-            data-testid={dataTestId.registrationLandingPage.npi}
-            title={t('registration.description.npi_disciplines')}
-            content={`${t(`disciplines.${entityDescription.npiSubjectHeading}` as any)} (${t(`disciplines.${findParentSubject(disciplines, entityDescription.npiSubjectHeading)}` as any)})`}
-          />
-        )}
-        {publicationInstance &&
-          ((isJournal(publicationInstance.type) || isPeriodicalMediaContribution(publicationInstance.type)) &&
-          journalPublicationInstance ? (
-            <PublicPublicationInstanceJournal publicationInstance={journalPublicationInstance} />
-          ) : isBook(publicationInstance.type) ? (
-            <>
-              <PublicPublicationInstanceBook publicationInstance={publicationInstance as BookPublicationInstance} />
-              <PublicIsbnContent
-                isbnList={(registration as BookRegistration).entityDescription.reference?.publicationContext.isbnList}
-              />
-              <RevisionInformation revision={(publicationContext as BookPublicationContext).revision} />
-            </>
-          ) : isDegree(publicationInstance.type) ? (
-            <>
-              {(publicationContext as DegreePublicationContext)?.course?.code &&
-                (publicationInstance.type === DegreeType.Bachelor ||
-                  publicationInstance.type === DegreeType.Master ||
-                  publicationInstance.type === DegreeType.Other) && (
-                  <PublicPageInfoEntry
-                    title={t('registration.resource_type.course_code')}
-                    content={(publicationContext as DegreePublicationContext).course?.code}
+        <Box component="dl">
```

Comment:
Burde ikke trenge Box her om du ikke skal ha noen styling. Ligger kanskje igjen fra tidligere utprøving?
```suggestion
        <dl>
```